src/Components/Footer/Footer.scss

```
@@ -1,2 +1,71 @@
-@import "../../Styles/common.scss";
-@import "../../Styles/reset.scss";
+@import '../../Styles/common.scss';
+@import '../../Styles/reset.scss';
+
+.Footer {
+  width: 100%;
+  position: fixed;
+  bottom: 0;
+  padding-bottom: 50px;
+  background-color: $footerbot;
+  .footer-top {
```

```suggestion
  background-color: $footerbot;
  
  .footer-top {
```
- 새로운 셀렉터 위에는 한줄 띄워주세요.
- 변수명, 클래스명은 카멜케이스로 지어주세요
- 색상에 대한 변수명이 footer bottom은 어색합니다. 색상명 뿐 아니라 클래스명 포함하여 변수명에 top, bottom, first, second 등 방향, 순서 등을 기입하는 건 좋지 않은 습관입니다. 보다 구체적인 내용을 기반으로 이름을 지어주세요.

---

+ 새로운 셀렉터와 이전 세렉터 사이에 한 행 추가했습니다.
+  css 변수명 카멜케이스로 바꿨습니다.
ex) $kakaoYellow
+  css 변수명 조금더 목적이 뚜렷하게 개명했습니다.
ex) $footerUpperColor  / $footerBottomColor